src/lib/MapOfTheWeekSection.svelte

```diff
@@ -41,6 +80,18 @@
             {/if}
           </p>
           <p class="review">{mapOfTheWeek.review}</p>
+          <div class="action-bar">
+            {#if mapOfTheWeek.showcase != null}
+              <button class="open-showcase-button" on:click={() => openShowcase()}>
+                Watch the showcase
+              </button>
+            {:else}
+              <!--Putting a div here so the OneClick Download URL stays in place-->
+              <!--Either than conditionally injecting styles-->
+              <div />
```

Comment:
Smart! I guess the flex-direction: row-reverse wouldn't be too terrible for screenreaders here, since it doesn't change the semantics, but I've opted for the margin-left approach. I think that one is less confusing for whoever my come after and try to understand it.

I did add you as co-author, but I must have screwed it up, since  Github is not picking it up. Sorry!